itest/integration_test.go

```diff
@@ -0,0 +1,134 @@
+				"vulnerabilities.aquasecurity.github.io",
+			)))
+
+			_, err = kubernetesClientset.CoreV1().Namespaces().Get(context.TODO(), "starboard", meta.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			// TODO Assert other Kubernetes resources that we create in the init command
+		})
+	})
+
+	Describe("Running version command", func() {
+		It("should print the current version of the executable binary", func() {
+			command := exec.Command(pathToStarboardCLI, []string{"version"}...)
+			session, err := Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(session).Should(Say("Starboard Version: {Version:dev Commit:none Date:unknown}\n"))
+		})
+	})
+
+	Describe("Running kube-bench", func() {
+		It("should run kube-bench", func() {
+			command := exec.Command(pathToStarboardCLI, "kube-bench", "-v", "3")
+			session, err := Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(session, scanJobTimeout).Should(Exit(0))
+
+			nodeNames, err := GetNodeNames(context.TODO())
+			Expect(err).ToNot(HaveOccurred())
+
+			for _, nodeName := range nodeNames {
+				reportList, err := starboardClientset.AquasecurityV1alpha1().CISKubeBenchReports().List(context.TODO(), meta.ListOptions{
+					LabelSelector: labels.Set{
+						kube.LabelResourceKind: string(kube.KindNode),
+						kube.LabelResourceName: nodeName,
+					}.String(),
+				})
+				Expect(err).ToNot(HaveOccurred())
+				Expect(reportList.Items).To(HaveLen(1), "Expected CISKubeBenchReport for node %s but not found", nodeName)
+			}
+		})
+	})
+
+	// FIXME Figure out why kube-hunter is failing on GitHub actions runner, whereas it's fine with local KIND cluster
+	PDescribe("Running kube-hunter", func() {
+		It("should run kube-hunter", func() {
+			command := exec.Command(pathToStarboardCLI, "kube-hunter", "-v", "3")
+			session, err := Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(session, scanJobTimeout).Should(Exit(0))
+
+			report, err := starboardClientset.AquasecurityV1alpha1().KubeHunterReports().
+				Get(context.TODO(), "cluster", meta.GetOptions{})
+
+			Expect(err).ToNot(HaveOccurred())
+			Expect(report.Labels).To(MatchAllKeys(Keys{
+				kube.LabelResourceKind: Equal("Cluster"),
+				kube.LabelResourceName: Equal("cluster"),
+			}))
+		})
+	})
+
+	AfterEach(func() {
+		command := exec.Command(pathToStarboardCLI, []string{"cleanup", "-v", "3"}...)
+		session, err := Start(command, GinkgoWriter, GinkgoWriter)
+		Expect(err).ToNot(HaveOccurred())
+		Eventually(session).Should(Exit(0))
+
+		// TODO We have to wait for the termination of the starboard namespace. Otherwise the init command
+		// TODO run by the BeforeEach callback fails when it attempts to create Kubernetes objects in the
+		// TODO starboard namespace that is being terminated.
+		//
+		// TODO Maybe the cleanup command should block and wait unit the namespace is terminated?
```

Comment:
I think this might be a good idea for users too. I ran into this recently - cleanup followed quickly by init failed. It was pretty obvious why, but waiting for the namespace to be gone would prevent it from being a problem at all.

Reply:
Created a task to follow up https://github.com/aquasecurity/starboard/issues/99